### PR TITLE
Add hbreader package

### PR DESCRIPTION
**Pull request by dstansby**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/hbreader/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hbreader-{{ version }}.tar.gz
+  sha256: d2c132f8ba6276d794c66224c3297cec25c8079d0a4cf019c061611e0a3b94fa
+  patches:
+    - tests.patch
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win or py<37]
```
Comment (synapticarbors) [inline recipes/hbreader/meta.yaml:17]: If this project isn't building any c-extensions, can it be a `noarch: python` build?
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

Then you can drop the `skip` and then just change the python requirement under `host` and `run` to `python >=3.7`

Comment (dstansby) [reply recipes/hbreader/meta.yaml:17]: The motivation here was that the windows tests are failing, but it's still worth releasing macOS/linux build for now.

**Comment by dstansby** [comment]
@conda-forge/help-python, ready for review!

**Comment by synapticarbors** [comment]
Last I looked at this, I raised a question about this being `noarch: python`. Could you either switch it or let me know why it shouldn't be? I know that you said that the windows test were failing, but I can't see what was causing them to fail. In the upstream source repository, I can't see any indication that it shouldn't run on Windows. If it's just a failing test (because the test has been mis-specified), but we believe the package should work, then we shouldn't drop the windows build.

**Pull request closed without merging**